model/player.go

```diff
@@ -5,17 +5,19 @@ package model
 
 import (
 	"encoding/json"
-	"golang.org/x/crypto/bcrypt"
 	"io"
 	"regexp"
 	"strings"
+
+	"golang.org/x/crypto/bcrypt"
 )
 
 const (
 	DEFAULT_LOCALE      = "en"
 	MIN_PASSWORD_LENGTH = 5
 	MIN_USERNAME_LENGTH = 4
 	MAX_USERNAME_LENGTH = 24
+	ADMIN_USERNAME      = "admin"
```

Comment:
The user object should have a flag indicating whether they're an admin or not. This feels so dirty, man.